core/src/main/scala/cats/instances/bitSet.scala

```diff
@@ -1,8 +0,0 @@
-package cats.instances
-
-import scala.collection.immutable.BitSet
-import cats.Show
-
-trait BitSetInstances extends cats.kernel.instances.BitSetInstances {
```

Comment:
There are two other instances from `kernel` provided in `cats.implicits` through this.